homeassistant/components/sensor/velo.py

```diff
@@ -0,0 +1,105 @@
+"""
```

Comment:
We should rename the platform to something `velo_antwerpen` as `velo` is a bit to generic.